_opensearch/rest-api/nodes-apis/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-title: Nodes API
+title: Nodes APIs
```

Comment:
We should use either "Node APIs" or "Nodes API" for grammatical purposes. Since this does encompass multiple endpoints, let's use "Node APIs".

Reply:
I am fine with that, this also means that we need to update `parent` field in all child pages.

Reply:
Fair point on the REST API. I'm fine with it matching and not worrying about the language aspect if you are. :D

Reply:
Yes, I think so.
The redirect should be IMO discussed as a general topic for the documentation.